app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+  after_save :update_likes_counter
+
+  def self.update_likes_counter(post)
```

Comment:
**[OPTIONAL]** Good job using the the Rails Active Callbacks `after_save` to call the `#update_likes_counter` 👍 here you can also make the method `private` by adding `private` key word at line 5. And you can do the same for post and comment models.